workers/unity/Packages/com.improbable.gdk.tools/ScriptableGdkToolsConfiguration.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public class ScriptableGdkToolsConfiguration : ScriptableObject
+    {
+        public string SchemaStdLibDir = DefaultValues.SchemaStdLibDir;
+        public List<string> SchemaSourceDirs = new List<string> { DefaultValues.SchemaSourceDir };
+        public string CodegenOutputDir = DefaultValues.CodegenOutputDir;
+
+        private const string AssetPath = "Assets/Config/GdkToolsConfiguration.asset";
+
+        internal List<string> Validate()
+        {
+            var errors = new List<string>();
+            if (string.IsNullOrEmpty(SchemaStdLibDir))
+            {
+                errors.Add($"{GdkToolsConfigurationInspector.SchemaStdLibDirLabel} cannot be empty!");
```

Comment:
Remove "!" from error messages. Keep 'em dry and informative.
cc @ElleEss for more guidance.